benchmarks/shootout/test_on_sgx.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+benchmarks = [
+    "base64",
+    "fib2",
+    "gimli",
+    "heapsort",
+    "matrix",
+    "memmove",
+    "nestedloop",
+    "nestedloop2",
+    "nestedloop3",
+    "random",
+    "seqhash",
+    "sieve",
+    "strchr",
+    "switch2",
+]
+
+
+def main():
+
+    os.system("./build_iwasm_sgx.sh")
+
+    for benchmark in benchmarks:
+        ret_code = os.system(
+            "./test_on_sgx.sh {bench} > {bench}.txt".format(bench=benchmark)
```

Comment:
Dumping to {bench.txt} makes no output on screen, had better remove `> {bench}.txt` here and add it in some lines of test_on_sgx.sh?